src/libraries/Microsoft.Extensions.Hosting.Systemd/tests/UseSystemdTests.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting.Internal;
+using Xunit;
+
+namespace Microsoft.Extensions.Hosting
+{
+    public class UseSystemdTests
+    {
+        [Fact]
+        public void DefaultsToOffOutsideOfService()
```

Comment:
Man, these aren't great tests :(. But I guess that is what exists today, so that's what we should use.

Reply:
https://github.com/dotnet/runtime/issues/49725 just to track if we want to add more tests.